tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.cs

```diff
@@ -187,11 +192,20 @@ private static void TryInvokeManagedMethod(string typeName, string methodName, s
         private static bool ReadBooleanEnvironmentVariable(string key, bool defaultValue)
         {
             var value = ReadEnvironmentVariable(key);
+            if (value == null)
+            {
+                return defaultValue;
+            }
+
+            return Parse(value);
+        }
+
+        private static bool Parse(string value)
+        {
             return value switch
             {
                 "1" or "true" or "True" or "TRUE" or "t" or "T" => true,
-                "0" or "false" or "False" or "FALSE" or "f" or "F" => false,
-                _ => defaultValue
```

Comment:
I'm not sure we should change this, as it's different to both how it works elsewhere in the tracer and in native code? 😕

Reply:
Good catch: reverting